awscli/examples/grafana/create-workspace.rst

```diff
@@ -0,0 +1,55 @@
+**To create a Grafana workspace**
+
+The following ``create-workspace`` example creates a new Amazon Managed Grafana workspace with basic configuration. ::
+
+    aws grafana create-workspace \
+        --workspace-name "MyGrafanaWorkspace" \
+        --workspace-description "Development team monitoring workspace" \
+        --account-access-type CURRENT_ACCOUNT \
+        --authentication-providers AWS_SSO \
+        --permission-type SERVICE_MANAGED
+
+Output::
+
+    {
+        "workspace": {
+            "id": "g-1234567890",
+            "name": "MyGrafanaWorkspace",
+            "description": "Development team monitoring workspace",
+            "endpoint": "https://g-1234567890.grafana-workspace.us-east-1.amazonaws.com",
+            "status": "CREATING",
+            "created": "2023-05-07T20:21:01.656000+00:00",
+            "modified": "2023-05-07T20:21:01.656000+00:00",
+            "accountAccessType": "CURRENT_ACCOUNT",
+            "authenticationProviders": ["AWS_SSO"],
+            "permissionType": "SERVICE_MANAGED",
+            "grafanaVersion": "9.4"
+        }
+    }
+
```

Comment:
Please provide a "For more information" link for every example in this request. See format in the contribution guide: https://aws.github.io/aws-cli/docs_styleguide.html#for-more-information-link